dart/lib/src/protocol/debug_image.dart

```diff
@@ -11,32 +11,32 @@ import 'package:meta/meta.dart';
 /// more details : https://develop.sentry.dev/sdk/event-payloads/debugmeta/
 @immutable
 class DebugImage {
```

Comment:
DebugImage has a couple fields which are required. Should I make them non-nullable?

Reply:
I think only `type` should be required as the requirements change based on the type
what we could do is to offer named ctors so we could use non-nullable field based on the ctor (maybe lets not do it now).
all the fields should be nullable though, to support each type of image (based on type)